### PR TITLE
Tweak: Converting into cult also purge all holy water from body

### PR DESCRIPTION
**Pull request by ErdGinalD**
## What Does This PR Do
Clear holywater from new body during converting process

## Why It's Good For The Game
some CLEVER AS F..REAKING ENSTEIN players fill their stomach with holy water and go fight to the cult, KNOWING they are not be converted. And then accidentaly "forgot" about 300 liters of cursed liquid into their bodyes after converting.
i need to fix that

## Images of changes
image

## Testing
copypasta from cult dagger

## Changelog
:cl:
tweak: Converting into cult also purge all holy water from body
/:cl:

### CODE REVIEW

**Review by Fordoxia — APPROVED** [review]
Looks good to Nar'Sie!

**Review by Oyu07 — APPROVED** [review]
chaplains hate this one trick

**Review by PopeDaveThe3th — APPROVED** [review]

File: code/game/gamemodes/cult/runes.dm [hunk 1]
```diff
@@ -356,6 +356,8 @@ structure_check() searches for nearby cultist structures required for the invoca
 
 		H.uncuff()
 		H.Silence(6 SECONDS) //Prevent "HALP MAINT CULT" before you realise you're converted
+		if(H.reagents && H.reagents.has_reagent("holywater"))
```
Comment (Contrabang) [inline code/game/gamemodes/cult/runes.dm:359]:
```suggestion
		if(H.reagents?.has_reagent("holywater"))
```

**Review by Henri215 — APPROVED** [review]
Lewcs good to Oyu

**Review by Contrabang — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**